app/controllers/bookings_controller.rb

```diff
@@ -1,74 +1,60 @@
 class BookingsController < ApplicationController
-  before_action :set_booking, only: [:show, :edit, :update, :destroy]
 
-  # GET /bookings
-  # GET /bookings.json
   def index
-    @bookings = Booking.all
+    if current_admin.nil?
+      @bookings = Booking.includes(:city, :cleaner, :customer).where(customer_id: session[:customer_id])
+    else
+      redirect_to admins_path
+    end
   end
 
-  # GET /bookings/1
-  # GET /bookings/1.json
   def show
   end
 
-  # GET /bookings/new
   def new
-    @booking = Booking.new
+    if current_admin.nil?
+      @booking = Booking.new
+      @cities = City.all
+    else
+      redirect_to admins_path
+    end
   end
 
-  # GET /bookings/1/edit
   def edit
+    @city_cleaner = CitiesCleaner.where(city_id: params[:selectedCity]).pluck(:cleaner_id)
```

Comment:
Make it @city_cleaner_id as it stores id of the cleaner. Give relevant names to instance variables!